kahuna/public/js/components/gr-title/gr-title.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import 'titip';
+
+export const module = angular.module('grTitle', []);
+
+module.directive('grTitle', [function () {
+    return {
+        restrict: 'A',
+        link: function (scope, element, attrs) {
+            const theme = attrs.grTitleTheme || 'default';
+            const position = attrs.grTitlePosition || 'bottom';
```

Comment:
`position` is used [here](https://github.com/guardian/grid/pull/1573/files#diff-f858b9ead9e730dc77b95579832a991aR38)

Was actually debating hardcoding `theme` - will do so!